lib/ui/view/channel/channel_view/channel_page_viewmodel.dart

```diff
@@ -6,11 +6,13 @@ import 'package:stacked_services/stacked_services.dart';
 class ChannelPageViewModel extends BaseViewModel {
   final _navigationService = locator<NavigationService>();
 
-  navigateToChannelInfo() {
-    _navigationService.navigateTo(Routes.channelInfoView);
+  Future navigateToChannelInfo() async{
+   await _navigationService.navigateTo(Routes.channelInfoView);
```

Comment:
why "awaiting"?

Reply:
that's what i did to make my implementation work